src/app/Dashboard/components/DashboardComponent.js

```diff
@@ -83,11 +83,11 @@ export class DashboardComponent extends Component {
                 <div className='user-dashboard__actions col-sm-12'>
                   <h3 className='user-dashboard__title'>My Activities</h3>
                   <div>
-                    <ButtonComponent className='button__add'>
+                    <ButtonComponent className='user-dashboard__button button__add'>
```

Comment:
Can we refactor this component, we can have the span(s) inside the Button component and have the name passed as props and Icon. but we can do that in another PR.
And change the name of the component to Button

Reply:
The reason is